packages/vite/src/node/importGlob.ts

```diff
@@ -59,7 +60,7 @@ export async function transformImportGlob(
   }
   const files = glob.sync(pattern, {
     cwd: base,
-    ignore: ['**/node_modules/**']
+    ignore: ['**/node_modules/**', ...(ignore ? [ignore] : [])]
```

Comment:
Should we also overwrite the default `ignore` option if the user explicit set `ignore` option. In that case, we could also close #1903 by `import.meta.glob('...', { ignore: [] })`.

Reply:
The other solution of #1903 would be something like `import.meta.glob('...', { node_modules: true })`

Reply:
Ok, I will implement it later.

Reply:
I think we should leave a `ignoreNodeModules` option (like your second solution) to support this, otherwise, the user will have to manually add the node_modules item every time the `ignore` option is used.

Reply:
Yes, I'm also prefer option like `import.meta.glob('...', { node_modules: true })`. We don't need to solve #1903 in this PR. Maybe it's better to let the vite team consider weather should and how to support the feature in #1903.